pkg/oc/cli/admin/policy/subject_review.go

```diff
@@ -43,16 +41,27 @@ var (
 	$  %[1]s -f myresourcewithsa.yaml `)
 )
 
-const SubjectReviewRecommendedName = "scc-subject-review"
+const (
+	SubjectReviewRecommendedName = "scc-subject-review"
+
+	sccSubjectReviewTabWriterMinWidth = 0
+	sccSubjectReviewTabWriterWidth    = 7
+	sccSubjectReviewTabWriterPadding  = 3
+	sccSubjectReviewTabWriterPadChar  = ' '
+	sccSubjectReviewTabWriterFlags    = 0
```

Comment:
These consts are identical for all commands, but yet we define all of them in each file. Make one set and re-use everywhere.